report-viewer/src/App.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <router-view/>
+</template>
+
+<style>
+@font-face {
+  font-family: "JetBrains Mono";
+  src: url("./fonts/jetbrains-mono/JetBrainsMono-Light.ttf");
```

Comment:
What is the license of the fonts you've added to the fonts folder?

Reply:
I think it has the [[SIL Open Font License 1.1](https://github.com/JetBrains/JetBrainsMono/blob/master/OFL.txt)](https://www.jetbrains.com/lp/mono/#license).

Reply:
It has SIL Open Font Licence. See https://www.jetbrains.com/lp/mono/#license.